README.md

```diff
@@ -30,7 +30,7 @@ yarn add @itwin/itwinui-react
 
 ---
 
-## Use
+## Usage
```

Comment:
if you change here, then change install -> installation too..
Why use is wrong?

Reply:
Idk, I've never seen "Use" in popular libraries. I've mostly seen "Usage" and "Examples".

But I've seen both "Install" and "Installation".